src/watch.jl

```diff
@@ -68,22 +67,24 @@ function init_revise!()
     @eval (@__MODULE__) using Revise
 end
 
-_report_and_watch_file(args...; kwargs...) = _report_and_watch_file(stdout::IO, args...; kwargs...)
-function _report_and_watch_file(io::IO, filename::AbstractString; jetconfigs...)
+function _report_and_watch_file(filename::AbstractString; jetconfigs...)
     config = WatchConfig(; jetconfigs...)
 
-    included_files, _ = report_file(io, filename; jetconfigs...)
+    res = report_file(filename; jetconfigs...)
+    display(res)
+    included_files = res.res.included_files
 
     interrupted = false
     while !interrupted
         try
             Revise.entr(collect(included_files), config.revise_modules;
                         postpone = true, all = config.revise_all) do
-                println(io)
-                included_files′, _ = report_file(io, filename; jetconfigs...)
-                if any(∉(included_files), included_files′)
+                res = report_file(filename; jetconfigs...)
+                display(res)
```

Comment:
@pfitzseb do think this `display` call is reasonable ? In watch mode, we don't return each result of `report_file` to REPL.

Reply:
Hm, seems like it could be a bit annoying in the REPL?

Reply:
Well, the watch mode prints analysis result on every update, so the behavior is same as far as `display(res)` prints into `stdout` (I think it's really silly to print everything everytime, though)